src/main_chart.py

```diff
@@ -95,23 +95,59 @@ def get_main_chart_figure(
                  "JediSwap_debt_token_supply": "#4C83D0"}
 
     # TODO: Align colors with the rest of the app.
-    figure = plotly.express.bar(
-        data_frame=data,
-        x="collateral_token_price",
-        # TODO
-        y=["debt_token_supply", "liquidable_debt_at_interval"],
-        # y=[f"{amm}_debt_token_supply" for amm in amms] + ["liquidable_debt_at_interval"],
+    color_map_protocol = {"liquidable_debt_at_interval_zkLend": "#ff7f0e", # Orange
```

Comment:
could you please change colors to 3 distinguishable from one another shades of yellow.